### PR TITLE
For ES 2.3.1

### PR DESCRIPTION
**Pull request by tokikanno**
Update version info and readme for  ES 2.3.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +37,10 @@ Install
 
 copy and unzip `target/releases/elasticsearch-analysis-ik-{version}.zip` to `your-es-root/plugins/ik`
 
+**for ES > 2.0**
+
+`bin/plugin install file:/path/to/elasticsearch-analysis-ik-{version}.zip`
```
Comment (medcl) [inline README.md]: #173 目前不支持这种安装方式

Comment (tokikanno) [inline README.md]: Yes, I noticed that immediately after submit this PR.
This line is removed.

**Comment by JohnTian** [comment]
Why elasticsearch update so quickly?

**Comment by medcl** [comment]
@JohnTian Good question, there are some several reasons, elasticsearch is  time-based release, which means every three week (~three week), there will be new minor release, ship with improvements,bugfix and new features, ship fast and get feedback quickly.

**Pull request merged**